ts/components/InstabugButtons.tsx

```diff
@@ -93,6 +94,7 @@ class InstabugButtonsComponent extends React.PureComponent<Props, State> {
             this.state.instabugReportType.value,
             dismiss
           );
+          this.props.dispatchUpdateInstabugBadge();
```

Comment:
```suggestion
          // when user dismisses instabug report (chat or bug) we update the unread messages counter. 
          // This is because user could have read or reply to some messages
          this.props.dispatchUpdateInstabugBadge();
```
Always leave a comment describing what is happening

Reply:
Ok :)